entrypoint.sh

```diff
@@ -139,6 +139,9 @@ if [[ -f "$GITHUB_WORKSPACE/$INPUT_IDE_VERSIONS" ]]; then
   echo "$INPUT_IDE_VERSIONS" | while read -r INPUT_IDE_VERSION; do
     gh_debug "                   => $INPUT_IDE_VERSION"
   done
+else
```

Comment:
I wonder if there is some check we could have to test if the input is (intended to be) a file and only print this message then. Would it make sense to see if the `$INPUT_IDE_VERSIONS` variable either ends in `.txt` and/or does *not* contain a `:` (used to separate the IDE and version)? I am open to any other ideas.

I would prefer to not have this message printed during a 'normal' path, as it might mislead or cause confusion for people. Hence my desire for a more specific check here.

What do you think?

Reply:
Yep, that sounds better. I've changed the `else` to `elif` and check whether the string doesn't contain a `:` before printing the message